libmpv/opengl_cb.h

```diff
@@ -151,6 +151,22 @@ extern "C" {
  * up until mpv_opengl_cb_uninit_gl() is called. If the name is not anything
  * you know/expected, return NULL from the function.
  *
+ * * Windowing system scaling
+ * ------------------------------------
+ *
+ * When using GL, sometimes GL rendering window is upscaled to display buffer.
+ * Typically with drm where GL framebuffer can be upscaled at later stage.
+ * In That case glMPGetNativeDisplay("opengl-cb-params") should return an
+ * mp_opengl_cb_params struct pointer :
+ */
+struct mp_opengl_cb_params {
```

Comment:
Rename it to `opengl-cb-window-pos` and `mpv_opengl_cb_window_pos`, and mention that this is an obscure mechanism for some hardware overlays.